src/components/ConnectToQuickbooksOnlineButton/index.native.tsx

```diff
@@ -36,6 +36,8 @@ function ConnectToQuickbooksOnlineButton({
     const webViewRef = useRef<WebView>(null);
     const [isWebViewOpen, setWebViewOpen] = useState(false);
     const {isOffline} = useNetwork();
+    const currentIntegration = translate('workspace.accounting.integrationName', integrationToDisconnect);
```

Comment:
Would be great if you can rollback this change. I've already added my changes based on the new copy.

Reply:
https://github.com/Expensify/App/pull/44218#issuecomment-2189593036